efficientdet/utils.py

```diff
@@ -46,6 +46,26 @@ def activation_fn(features: tf.Tensor, act_type: Text):
     raise ValueError('Unsupported act_type {}'.format(act_type))
 
 
+class Activation_fn(tf.keras.layers.Layer):
```

Comment:
This class might not be necessary because tf.keras already has a layer for activation functions.  It is used like this:

```
tf.keras.layers.Activation('swish')
tf.keras.layers.Activation(tf.sigmoid)
tf.keras.layers.Activation(tf.nn.relu6)
```
https://www.tensorflow.org/api_docs/python/tf/keras/layers/Activation

Reply:
just use tensorflow ops directly.

Reply:
I'm sorry @fsx950223 , I don't understand your comment, using tf.keras.layers.Activation constructed the same way as before should be good ? Or is there some flaws with using Activation ? Also, if you have times to review latest commit on names, I think I got them but you may have seen some errors. Thank you and @Ely-S, for your reviews btw, very appreciated.

Reply:
Yes, you should add a test case for check op names.